cmd/core-command/res/configuration.toml

```diff
@@ -1,11 +1,5 @@
 [Writable]
-LogLevel = "INFO"
```

Comment:
This needs to stay and removed from common. Same in other service configs.

We want to keep LogLevel in private configs so it can be changed for a single service rather than all.

Reply:
I thought we wanted LogLevel to be in common config so that it only has one place it needs to be changed.

Reply:
Nope, since that will change is for all services. There was discussion and we thought that would not be a go thing and that changing it should be a on a service by service basis.